packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -300,6 +300,10 @@ const BlockInspectorSingleBlock = ( { clientId, blockName } ) => {
 						label={ __( 'Border' ) }
 					/>
 					<InspectorControls.Slot group="styles" />
+					<InspectorControls.Slot
+						group="media"
+						label={ __( 'Media' ) }
```

Comment:
This is most likely overthinking it, but I was wondering if this might be confused with the other "Media", the library? Since it's specific to backgrounds for now, what about "Background media"?

Reply:
Good question! For the panel title, I was thinking of trying to keep it generic to `Media` for consistency with the Site Logo block that already has a Media panel, and leave the specifics to the individual block support within it. The idea being that we then have a consistent panel across any block that has media, irrespective of where that media winds up being output.

That was just my initial thinking though, happy to change things around! For now, I might leave it as-is while we await further design feedback 🙂

Reply:
Sounds good to me